vmm/src/default_syscalls.rs

```diff
@@ -486,6 +510,10 @@ pub fn default_context() -> Result<SeccompFilterContext, Error> {
                 libc::SYS_stat,
                 (0, vec![SeccompRule::new(vec![], SeccompAction::Allow)]),
             ),
+            (
+                libc::SYS_tkill,
+                (0, vec![SeccompRule::new(vec![], SeccompAction::Allow)]),
```

Comment:
What arguments does `tkill` get? I'm interested in the `sigid`. If we absolutely have to allow it, can we only let a specific `sigid` through?

Reply:
Ideally we would build this dynamically and even enforce the allowed PIDs. In the meantime I will look into filtering only specific `sigid`s.